### PR TITLE
Define and activate Azure 11.4.0 release

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#8877

Depends on:
- https://github.com/giantswarm/roadmap/issues/141
- https://github.com/giantswarm/azure-operator/pull/817

### CODE REVIEW

File: release-notes/azure/v11.3.1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+This release changes NGINX Service external traffic policy from default `Cluster` to `Local`.
+
+### azure-operator v4.0.1
+
+- Use Local as NGINX LoadBalancer Service external traffic policy.
+
+### nginx-ingress-controller v1.6.10
+
+- Use Local as NGINX NodePort Service external traffic policy.
```
Comment (MartaD) [inline release-notes/azure/v11.3.1.md:11]:
```suggestion
- Use Local as NGINX NodePort Service external traffic policy.

### cluster-operator 0.23.9
```

File: azure.yaml [hunk 2]
```diff
@@ -3,9 +3,63 @@ kind: Release
...
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: azure-operator
+    version: 4.0.1-dev
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 0.23.8
```
[the remark this answers is not in the document]
Comment (sslavic) [reply azure.yaml:43]: even nginx app 1.6.10 was developed in mind that Azure is "legacy", so this bump alone is likely not enough, we may need to release another nginx release, if not also cluster-operator (legacy branch); will sync with @whites11

**Comment by axbarsan** [comment]
Removed `nginx-ingress-controller` from the `11.4.0` app list since it should be optional from now on.

**Comment by sslavic** [comment]
@axbarsan any reason why nginx app removal from 11.4.0 release definition was not actually reverted?

**Comment by axbarsan** [comment]
> @axbarsan any reason why nginx app removal from 11.4.0 release definition was not actually reverted?

I forgot to push. Thank you for noticing 👊

**Pull request closed without merging**